Cargo.lock

```diff
@@ -2173,6 +2183,28 @@ dependencies = [
  "memchr",
 ]
 
+[[package]]
+name = "quickcheck"
+version = "1.0.3"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "588f6378e4dd99458b60ec275b4477add41ce4fa9f64dcba6f15adccb19b50d6"
+dependencies = [
+ "env_logger 0.8.4",
+ "log",
+ "rand",
+]
```

Comment:
Do we make use of the `env_logger` and `regex` feature or could we disable all default features? 

https://docs.rs/crate/quickcheck/1.0.3/features